src/app/fdctl/configure/workspace.c

```diff
@@ -21,6 +21,31 @@ init_perm( security_t *     security,
   check_res( security, NAME, RLIMIT_MEMLOCK, limit, "increase `RLIMIT_MEMLOCK` to lock the workspace in memory" );
 }
 
+static inline uchar *
+read_key( char const * key_path,
+          uchar      * key       ) {
+  FILE * key_file = fopen( key_path, "r" );
+  if( FD_UNLIKELY( !key_file ) ) {
+    if( FD_UNLIKELY( errno == ENOENT ) ) FD_LOG_ERR((
+          "The [consensus.identity_path] in your configuration expects a "
+          "keyfile at %s but there is no such file. Either update the "
+          "configuration file to point to your validator identity "
+          "keypair, or generate a new validator identity key by running "
+          "`fdctl keygen`", key_path ));
+    FD_LOG_ERR(( "Opening key file (%s) failed (%i-%s)", key_path,  errno, fd_io_strerror( errno ) ));
+  }
+
+  if( FD_UNLIKELY( 1!=fscanf( key_file, "[%hhu", &key[0] ) ) )   FD_LOG_ERR(( "parsing key file failed at pos=0"      ));
```

Comment:
Can you make this error message usable by a validator operator as well? Something like

```Error while parsing the validator identity key file at path `%s` specified by [consensus.identity_path] in your configuration. Solana key files should be formatted as a json array with 64 elements. The specific error was %s.```

Reply:
(Same for the below parsing errors. These will save operators a lot of time IMO.)

Reply:
Good idea. Added.